### DISCUSSION TITLE
Player headtracking implementation possible?

### DISCUSSION THREAD

**Discussion by czarnyafgan** [Try/OpenGothic#186] [Ideas]
Hi. Is it possible for you to implement basic player headtracking when in interaction with NPC's ? 

This would add a lot of immersion to game!

Please look at:

https://youtu.be/FSpcoqjQGcc

**Comment by Try** [comment]
In principle stuff like this should be handled by mod, not be engine itself.
On engine side there is a script api for it:
```c++
Ai_LookAt
Ai_StoplookAt
```
